README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/r-bioinformatics/edgePy.svg?branch=master)](https://travis-ci.org/r-bioinformatics/edgePy)
 [![codecov](https://codecov.io/gh/r-bioinformatics/edgePy/branch/master/graph/badge.svg)](https://codecov.io/gh/r-bioinformatics/edgePy)
 [![PyPI Version](https://badge.fury.io/py/edgePy.svg)](https://pypi.org/project/edgePy)
-[![GitHub License](https://img.shields.io/github/license/r-bioinformatics/edgePy.svg)](https://github.com/r-bioinformatics/edgePy/blob/master/LICENSE)
+[![GitHub License](https://img.shields.io/github/license/mashape/apistatus.svg)](https://github.com/r/bioinformatics/edgePy/blob/master/LICENSE)
```

Comment:
Hi @julianstanley you should leave this as-is. Don't edit this line.

What `img.shields.io` is doing is pinning our 'current' license.

This badge will automatically update once this PR is merged. Cool huh?

Reply:
That is indeed very cool. Thank you for pointing that out! I changed my branch's readme back to the original.